test/functional/change_stream.test.js

```diff
@@ -2800,7 +2815,7 @@ describe('Change Streams', function() {
             case 2:
               // only events after this point are relevant to this test
               events = [];
-              triggerResumableError(changeStream, () => events.push('error'));
+              triggerResumableError(changeStream, () => events.push('error'), 0);
```

Comment:
However, in this case the resumable error _needs_ to be instant because of how the test is construct; it has to trigger before the next `change` event is processed.